go.mod

```diff
@@ -55,7 +55,7 @@ require (
 	github.com/spf13/cobra v1.7.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
-	github.com/tendermint/tendermint v0.34.28
+	github.com/tendermint/tendermint v0.35.9
```

Comment:
???

Reply:
??? Also dont know how this happened, thx